testdata/pods/ocp10031/pod.json

```diff
@@ -10,7 +10,7 @@
   "spec": {
       "containers": [{
         "name": "hello-pod",
-        "image": "quay.io/openshifttest/hello-openshift-getent"
+        "image": "quay.io/openshifttest/hello-sdn@sha256:d5785550cf77b7932b090fcd1a2625472912fb3189d5973f177a5a2c347a1f95"
```

Comment:
Do you have to provide the sha? Would it make more sense to use 'latest' tag if needed instead?

Reply:
@asood-rh if you look at the existing yaml/json files they are using `sha`